### PR TITLE
Fix composer based config symlink

### PR DESCRIPTION
**Pull request by marfillaster**
**Note:** Please fill out the PR Template to ensure proper processing.

Closes: #

## Summary

**[Using SimpleSAMLphp with Shibboleth SSO](https://pantheon.io/docs/shibboleth-sso)**

## Effect
(Use this section to detail the changes summarized above, or remove if not needed)
The following changes are already committed:

-
-

## Remaining Work
(Remove if not needed)
The following changes still need to be completed:

- [X] Test environment https://dev-ken-d8-simplesaml.pantheonsite.io/simplesaml/

## Post Launch

**Do not remove** - To be completed by the docs team upon merge:

- [ ] Redirect `/docs/old-path/` => `/docs/new-path/` (if applicable)
- [ ] Include/exclude pages ^ respectively within docs search service provider (if applicable)
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by alexfornuto** [comment]
@kyletaylored you recently went through all these paths, can we get your opinion on this?

File: source/content/shibboleth-sso.md [hunk 1]
```diff
@@ -62,7 +62,7 @@ Commands below require a [nested docroot](/nested-docroot) structure and should
  composer require simplesamlphp/simplesamlphp
  ```
 
-1. Add a symlink from `web/simplesaml` to `vendor/simplesamlphp/simplesamlphp/www`:
+2. Add a symlink from `web/simplesaml` to `vendor/simplesamlphp/simplesamlphp/www`:
```
Comment (alexfornuto) [inline source/content/shibboleth-sso.md:65]: Each number should be 1. Markdown increments, and this way we can add or remove steps easily.

**Comment by kyletaylored** [comment]
After working through this with a customer this week, we might want to add another note / step about adding a metadata folder (similar to adding a config from config templates). This is also a step mentioned in the SimpleSAML module documentation.

**Comment by kyletaylored** [comment]
Update: added notes about optional metadata requirements.

**Comment by alexfornuto** [comment]
Merged in #5915, shown as conflicting due to rebasing for updates. @marfillaster make sure to branch off from the main repo next time, so I can make edits directly.

**Pull request closed without merging**